<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AvlWidget</class>
 <widget class="QWidget" name="AvlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="initBtn">
         <property name="text">
          <string>init_avl</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="inputKey"/>
       </item>
       <item>
        <widget class="QPushButton" name="addBtn">
         <property name="text">
          <string>add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delBtn">
         <property name="text">
          <string>remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchBtn">
         <property name="text">
          <string>search</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
